fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -1931,20 +1772,16 @@ ACTOR Future<Standalone<VectorRef<KeyValueRef>>> decodeRangeFileBlock(Reference<
 			StringRef decryptedData =
 			    wait(EncryptedRangeFileWriter::decrypt(cx, encryptHeader, dataPayloadStart, dataLen, &results.arena()));
 			reader = StringRefReader(decryptedData, restore_corrupted_data());
-			wait(decodeKVPairs(&reader, &results, true, encryptMode, blockDomainId, tenantCache));
+			wait(decodeKVPairs(&reader, &results, true, encryptMode, blockDomainId));
```

Comment:
if `decodeKVPairs` no long has `wait`, we can remove wait here and change the function to non-actor one.